frontend/scripts/react-components/nav/global-search/global-search.component.jsx

```diff
@@ -138,30 +135,48 @@ export default class GlobalSearch extends Component {
                   />
                 </div>
                 {!isEmpty(searchTerm) && (
-                  <ul className="search-results">
-                    {searchResults.slice(0, MAX_SEARCH_RESULTS).map((item, row) => (
-                      <GlobalSearchResult
-                        key={row}
-                        value={searchTerm}
-                        isLoading={isLoading}
-                        isHighlighted={row === highlightedIndex}
-                        item={item}
-                        itemProps={getItemProps({ item })}
-                        testId={`global-search-result-${kebabCase(item.name)}-${
-                          item.contextId
-                        }`.toLowerCase()}
-                      />
-                    ))}
-                    {noResults && (
-                      <li className="c-search-result -no-highlight">
-                        <div className="search-node-text-container">
-                          <span className="search-node-name" data-test="global-search-no-result">
-                            No results found
-                          </span>
-                        </div>
-                      </li>
-                    )}
-                  </ul>
+                  <FixedSizeList
+                    height={345}
+                    width={window.innerWidth}
+                    itemSize={85}
+                    itemData={searchResults}
+                    itemCount={searchResults.length}
+                    innerElementType="ul"
+                    className="search-results"
+                  >
+                    {({ index, style, data }) => {
+                      const item = data[index];
+
+                      if (noResults) {
+                        return (
+                          <li style={style} className="c-search-result -no-highlight">
+                            <div className="search-node-text-container">
+                              <span
+                                className="search-node-name"
+                                data-test="global-search-no-result"
+                              >
+                                No results found
```

Comment:
Not sure why but the "Not results" is not showing on the global results search. Could you check this?
![image](https://user-images.githubusercontent.com/9701591/73169520-88d7c600-40fc-11ea-8fb5-a8c2fad436e7.png)